app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,11 @@
+class TweetsController < ApplicationController
+    register Sinatra::ActiveRecordExtension
+    set :views, Proc.new { File.join(root, "../views/") }
+
+    configure do
+      enable :sessions
+      set :session_secret, "secret"
+    end
+
+
+end
```

Comment:
These configurations are set up in ApplicationController, so all we need here are the routes